Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: ctlogs
```

Comment:
Would it make sense to move this Makefile into the gen-known-logs directory? The issue being that people might assume that "make" should build certigo, which is false.